Demo/Demo/SwiftUIComponents/CommonComponents/LabelViewText.swift

```diff
@@ -0,0 +1,18 @@
+import SwiftUI
+
+struct LabelViewText: View {
```

Comment:
Open to alternate names for this `struct` as well. Right now it's only used in our result views but we could potentially use it in other views.

Reply:
FWIW on Android we have a similar `PropertyView`.

Reply:
I like this LabelViewText!